add processing of finngen

RobinM-code opened the pull request:
I added finngen processing.

(The changes to top_loci_by_chrom can be reverted since we don't need top_loci_by_chrom)

It would be great if you could have a look and let me know what you think!

This PR complements this PR from otg-data-loading: https://github.com/thehyve/otg-data-loading/pull/86

RobinM-code committed:
add processing of finngen

romanhaa commented on finngen/run_finngen_pipeline.sh:
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```
Do you think this script will take a while and does it print a lot to stdout? If it has no progress indicator and takes a long time, I think it may be useful to a log message with a timestamp before starting each script, just so we have an idea where we are at. But that's really only useful if it takes a long time. I made a small commit to inject those kinds of messages for the coloc step: https://github.com/thehyve/otg-data-loading/commit/52d1fc777b0435a32f1fe8a955e92d9bef49913c

RobinM-code replied:
I am not entirely sure since I only tested with 1 file but I think it really shouldn't take a lot of time. It's just reformatting some input to output that can be handled by the pipeline. Also, the formatting of this 1 file was finished way faster then the "real" finemapping step.

romanhaa replied:
Ok, then adding log statements would probably just produce noise and not much use.

romanhaa commented on finngen/1_make_finngen_finemapping_manifest.py:
```diff
@@ -7,79 +7,35 @@
     # Output directory for individual study fine-mapping results
-    output_path = 'output/'
-
-    keep_columns = [
```
Is this logic removed based on the format of the FinnGen studies that we were provided with? Those may have been processed for us (since we asked for a specific format) while EBI could have worked with unprocessed files. If that's the case, then it would probably be easier to not request format changes and work with original format as well. We should clarify that.

RobinM-code replied:
This "old" logic is based on the finngen manifest and some meta_data. This determines what finngen studies will be processed. I have set this up to process all finngen studies that are present in the pipeline run, which made more sense.  Also we really don't need `trait`, `trait_category`, `n_cases`, `n_controls` to format the top_loci and credset files, we only need `code`.

romanhaa replied:
Ok, sounds good.

romanhaa reviewed (commented):
Really nice. Not gonna lie, I can't go through all changes in detail but I trust that you tested the code.

RobinM-code committed:
remove partition top loci by chrom

RobinM-code committed:
create combine results script

Always run this script, not only after finngen.
This should be done to make sure the output will end up in the right dir

romanhaa commented:
Just a curiosity: Did you check whether the pipeline still runs fine even when there are no FinnGen studies?

RobinM-code commented:
> Just a curiosity: Did you check whether the pipeline still runs fine even when there are no FinnGen studies?

Very valid point! I had to fix some stuff in this commit: https://github.com/thehyve/otg-data-loading/pull/86/commits/072fd97476476ec1f0ce520a219746d3e859846c 

Thanks for pointing this out
